test/y2storage/partition_tables_test.rb

```diff
@@ -49,11 +51,38 @@
   end
 
   describe "#delete_all_partitions" do
+    subject(:ptable) { Y2Storage::Disk.find_by_name(fake_devicegraph, "/dev/sdb").partition_table }
+
     it "deletes all partitions in table" do
       # NOTE: it is important here to have partition table with logical partitions
       # as it need special handling, so test will cover it
       ptable.delete_all_partitions
       expect(ptable.partitions).to be_empty
     end
   end
+
+  describe "#unused_partition_slots" do
```

Comment:
I would maybe add some comment here explaining why we added this test. Something like:
```ruby
# This tests ensures the behavior of libstorage is stable in the future, because, in order
# to generate aligned partitions, the proposal code heavily relies on the fact that
# Y2Storage::PartitionTable#unused_partition_slots returns aligned regions.
```